arch/Kconfig

```diff
@@ -202,6 +202,14 @@ menu "Interrupt Configuration"
 #
 # Interrupt related configs
 #
+config DYNAMIC_INTERRUPTS
```

Comment:
Shouldn't this depend on "GEN_SW_ISR_TABLE"?

Reply:
no, this is a generic option. x86 doesn't use SW ISR table, it has a different implementation.